tests/test_trigger.py

```diff
@@ -63,3 +63,36 @@ def test_create_from_user_query(bot_fx):
         "target": "test_nick",
         "message": "Exceedingly smart test message."
     } in bot_fx.sent_messages
+
+def test_trigger_eq(bot_fx):
+    trigger1 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+    trigger2 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+
+    assert trigger1 == trigger1
+    assert trigger2 == trigger2
+
+    assert trigger1 == trigger2
+    assert trigger2 == trigger1
+
+def test_trigger_ne(bot_fx):
+    trigger1 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+    trigger2 = Trigger(bot_fx, ["some", "thing"], ["another thing", "thing"], "stupid_nick",
+                       "not_a_channel", "test_ident", "go.away")
+
+    assert trigger1 != trigger2
+    assert trigger2 != trigger1
+
+def test_trigger_hash():
+    trigger1 = Trigger(None, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+    trigger2 = Trigger(None, ["some", "thing"], ["some thing", "thing"], "test_nick",
```

Comment:
Methinks 
```py
trigger2 = deepcopy(trigger1)
```
would be a more effective solution, rather than a copy paste

Reply:
That would work, though these are just tests so performance isn't really an issue. And I believe this to be more verbose. Potatoes potahtoes though.